includes/pieces.h

```diff
@@ -35,6 +35,8 @@ class Piece {
 
     virtual void set();
 
+    virtual void draw();
```

Comment:
assuming this purposely a stub for now. I suggest you add TODOs to make reviewing this easier for me.